Modify `preserveOutliers` GraphQL API (#266)
aicers/review-web: CHANGELOG.md

- Modified the `Node`, `NodeProfile` fields.
  - Modified `Node` related CRUD APIs.
- Updated review-database to 0.29.1.

> - Changed GraphQL API `preserveOutliers` to use `PreserveOutlierOutput` in its response.

Author: 내용 수정하도록 하겠습니다.

- Modified the `Node`, `NodeProfile` fields.
  - Modified `Node` related CRUD APIs.
- Updated review-database to 0.29.1.

> - Modified GraphQL API
>   - Change response in the `preserve_outliers` GraphQL API from returning
>     `Result<usize>` to `Result<Vec<PreserveOutlierOutput>>`.

Reviewer: ```suggestion
- Changed GraphQL API `preserveOutliers` to use `PreserveOutlierOutput` in its response.
```

- GraphQL API `preserveOutliers` 가 응답에서 `PreserveOutlierOutput`을 사용하도록 변경했습니다.

를 deepl로 번역하여 위와 같은 결과를 얻었습니다. 위와 같은 변경을 제안드리고자 합니다.
